odoo/cli/db.py

```diff
@@ -123,15 +123,25 @@ def run(self, cmdargs):
         dump = subs.add_parser(
             "dump", help="Create a dump with filestore.",
             description="Creates a dump file. The dump is always in zip format "
-                        "(with filestore), to get a no-filestore format use "
-                        "pg_dump directly.")
+                        "(with filestore), to get another format, use "
+                        "dump_format argument.")
         dump.set_defaults(func=self.dump)
         dump.add_argument('database', help="database to dump")
         dump.add_argument(
             'dump_path', nargs='?', default='-',
             help="if provided, database is dumped to specified path, otherwise "
                  "or if `-`, dumped to stdout",
         )
+        dump.add_argument(
+            'dump_format', nargs='?', default='zip',
```

Comment:
We usually don't use optional positional arguments, btw can be a good idea to limit the available choices to what's supported
```suggestion
        dump.add_argument(
            '--format', dest='dump_format', choices=('zip', 'dump'), default='zip',
```